### DISCUSSION TITLE
proxy question

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#112] [Ideas]
does the proxy support 2 or more of the same channel but using 1 cxn back to the provider?

**Comment by euzu** [comment]
Each channel is handled separatly.
What do you mean with one connection back to provider?

**Reply by aniel300** [reply]
i have use and been using few proxy in the past months most of them have this feature where u are able to to tune into a given same channel from multiple clients but only use 1 cxn back to provider. am assuming there is some proxy/ ram cashing involve on this to make it happen.

**Comment by euzu** [comment]
now i understand your question, you are asking when multiple clients connect to m3u-filter and want to watch the same stream, there should be only one connection to the provider.

No this is not the case.

**Reply by aniel300** [reply]
yes. is there any way to implement this?

**Comment by euzu** [comment]
I currently.dont have a clue howto.implement this, because you need to track sessions and share connections. You need to track who watches and closes the stream and so on. Its not an easy task. You can create a feature request to keep this as todo.

**Reply by aniel300** [reply]
will do thanks.

**Reply by aniel300** [reply]
done: https://github.com/euzu/m3u-filter/issues/116
